src/android/src/main/java/io/radar/cordova/RadarCordovaPlugin.java

```diff
@@ -461,21 +471,90 @@ public void startTrip(final JSONArray args, final CallbackContext callbackContex
         final JSONObject optionsObj = args.getJSONObject(0);
 
         RadarTripOptions options = RadarTripOptions.fromJson(optionsObj);
-        Radar.startTrip(options);
+        Radar.startTrip(options,new Radar.RadarTripCallback() {
+            @Override
+            public void onComplete(@NonNull Radar.RadarStatus radarStatus) {
+                if (radarStatus == Radar.RadarStatus.SUCCESS) {
+                    callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.OK));
+                } else {
+                    callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.ERROR,radarStatus.toString()));
```

Comment:
@brettguenther @kochis Nit: Add a space between arguments

Reply:
We should also always use `PluginResult.Status.OK` and return `status` in a dict/obj like other endpoints for consistency. I'll make that change.